src/videoframeprocessor/DefaultVideoTransformDevice.ts

```diff
@@ -31,7 +32,11 @@ export default class DefaultVideoTransformDevice
     private processors: VideoFrameProcessor[],
     private browserBehavior: BrowserBehavior = new DefaultBrowserBehavior()
   ) {
-    this.pipe = new DefaultVideoFrameProcessorPipeline(this.logger, this.processors);
+    this.pipe = new DefaultVideoFrameProcessorPipeline(
+      this.logger,
+      this.processors,
+      new DefaultVideoFrameProcessorTimer()
```

Comment:
This explicit passing is not needed as you create one by default when instantiating.